packages/core/src/App/Containers/RealAccountSignup/__tests__/account-wizard.spec.tsx

```diff
@@ -188,26 +195,42 @@ describe('<AccountWizard />', () => {
     };
 
     it('should render AccountWizard component', () => {
-        render(<AccountWizard {...mock_props} />);
+        render(
+            <StoreProvider store={mock}>
+                <AccountWizard {...mock_props} />
+            </StoreProvider>
+        );
```

Comment:
why do we need add StoreProvider? 
For hooks, we should mock hook inside this unit test. right?